apps/client/src/app/App.tsx

```diff
@@ -19,8 +19,6 @@ import ViewAll from '../../../../libs/client/ui/src/lib/view-all/view-all.js';
 const Stack = createNativeStackNavigator();
 
 export const App = () => {
-  const [whatsNextYCoord, setWhatsNextYCoord] = useState<number>(0);
-  const scrollViewRef = useRef<null | ScrollView>(null);
```

Comment:
Why are these removed? I'm not sure what this PR is trying to do

Reply:
Those are leftovers from their original placeholder app that used to be there

Reply:
My messages are here

Reply:
Ok, and the unit test stuff that was commented out?

Reply:
Yeah beacuse there is not a proper unit test yet. At the moment, i'm working on getting it onto the emulator

Reply:
Ok